packages/contracts-bedrock/test/DeployOPChain.t.sol

```diff
@@ -414,31 +417,31 @@ contract DeployOPChain_Test is DeployOPChain_TestBase {
         // TODO Add fault proof contract assertions below once OPSM fully supports them.
 
         // Assert that individual input fields were properly set based on the input struct.
```

Comment:
A bunch of references to structs in this file that are outdated

Reply:
Removed in 1cbf77e1a